client/js/components/procedure/StatementSegmentsList/StatementSegment.vue

```diff
@@ -740,6 +740,7 @@ export default {
       this.$parent.$parent.resetSlidebar()
       this.toggleSlidebarContent({ prop: 'slidebar', val: { isOpen: true, segmentId: this.segment.id, showTab: 'map' } })
       this.$root.$emit('show-slidebar')
+      this.$root.$emit('segmentMap:show')
```

Comment:
I think we should try to avoid adding `$root.$emit`s to the code base since they are not supported anymore in vue 3. But since we haven't decided on a pattern to replace it yet, I don't consider this a blocker.